mteb/abstasks/AbsTask.py

```diff
@@ -224,10 +228,42 @@ def load_data(self, **kwargs):
         """
         if self.data_loaded:
             return
-        self.dataset = datasets.load_dataset(**self.metadata.dataset)  # type: ignore
+        if self.is_multilingual:
+            if self.fast_loading:
+                self.fast_load()
+            else:
+                self.dataset = {}
+                for hf_subset in self.hf_subsets:
+                    self.dataset[hf_subset] = datasets.load_dataset(
+                        name= hf_subset,
+                        **self.metadata.dataset,
+                    )
+        else:
+            self.dataset = datasets.load_dataset(**self.metadata.dataset)  # type: ignore
```

Comment:
```suggestion
            if self.fast_loading:
                self.fast_load()
            else:
                self.dataset = {}
                for hf_subset in self.hf_subsets:
                    self.dataset[hf_subset] = datasets.load_dataset( 
                        name= hf_subset,
                        **self.metadata.dataset,
                    )
```
I assume this should work in all cases?

Reply:
Yes, I think so

Reply:
Some monolingual datasets include the split name in their `metadata.dataset`, which causes this change to break them since `name=hf_subset` is passed. I'll roll back this change.